### ISSUE TITLE
"No Confirmation Needed" or "Confirmation Needed"

### ISSUE THREAD

**Issue by ryanleesipes** [thunderbird/appointment#460]
I like to confirm meeting times, but some people may not need to. We should expose an option in the short-term (and in the long-term expose this in onboarding) to be able to turn off confirmation (or we could choose to make that the default and have a "Confirmation Needed" option that you turn on). So that folks can allow people to grab time on their calendars instantly with no confirmation required.

**Comment by devmount** [comment]
I'll take care of that. Where should that "Confirmation Needed" option live? On the schedule page? Like e.g. here?

![image](https://github.com/thunderbird/appointment/assets/5441654/611962ad-0dc5-43d8-a79e-3a80307d06df)

cc @MelissaAutumn @solangevalverde

**Comment by malini** [comment]
I'll loop in design now, but let's just ship with something that we're okay with until then.

**Comment by malini** [comment]
@devmount I'm okay with that location for now, let's do it!

**Comment by devmount** [comment]
Alright, will do! Thank you!

**Comment by laurelterlesky** [comment]
@malini Can we reopen this? 

See the design solve here: https://www.figma.com/design/rDjrWQsgejQUyACjjXRwVa/TBA_2024_Q3_BookingConfirmationToggle?node-id=1-6&t=MnNACwn5bFQfifjV-1

cc: @solangevalverde

**Comment by MelissaAutumn** [comment]
I don't have access, but I've swapped this over to mvp as the feature is in, just not as designed..which I can't see 😅

**Comment by ryanleesipes** [comment]
Let's post images in when possible. Not just for Mel, but also so that contributors can follow along.

**Comment by devmount** [comment]
Thanks @ryanleesipes, totally agree! I'll take a look at this.

**Comment by laurelterlesky** [comment]
![Screenshot 2024-09-05 at 14 47 34](https://github.com/user-attachments/assets/0e0b7e20-e49b-45bf-a4d8-34895eb81066)

**Comment by laurelterlesky** [comment]
@devmount Do you need a dark mode mockup?

**Comment by devmount** [comment]
@laurelterlesky Yes, that would be great 🙏🏻

**Comment by laurelterlesky** [comment]
Ended up doing some updates so I could get to dark mode. Happy to walk you through them on Monday (tagged you in the figma file too)
![Screenshot 2024-09-06 at 15 56 07](https://github.com/user-attachments/assets/eaada8e4-b2cd-4fa4-9604-7886163d9960)

**Comment by laurelterlesky** [comment]
![Screenshot 2024-09-07 at 15 50 34](https://github.com/user-attachments/assets/5f7fc854-1c61-407e-8c73-9afbd2342b8b)

**Comment by MelissaAutumn** [comment]
Andreas implemented the new section and updates. I went in and tweaked the style to fit the font-size, and edgecases I ran into. Additionally I've added a max length of 16 characters, and the username will overflow into `melissa@th.../ <editable section>` if the username is too long.  

<img width="318" alt="image" src="https://github.com/user-attachments/assets/d62a337d-f26d-41b4-8252-81156a8f65b8">

**Comment by MelissaAutumn** [comment]
Oops just noticed there's some additional design tweaks of replacing the buttons with tbpro components. Can you convert the rest of the form to tbpro components @devmount ?

**Comment by devmount** [comment]
@MelissaAutumn Of course, I'll replace the buttons. We can add more tbpro elements (e.g. for those accordion cards) once we got that services-ui repo up an running!

**Comment by devmount** [comment]
Keeping this open since we still need to style:

- [ ] Checkbox
- [ ] Switch toggle
- [ ] Accordion cards (inactive, active)
